telemetry-schema.md

```diff
@@ -0,0 +1,71 @@
+## Attributes
+
+Key | Description
+--- | ---
+event | `perf-report` (can this take other values?)
+results | String holding a JSON-encoded `OuterResults`
+sentCount | string-encoded integer representing the number of `Results` in each `Test`
+
+## OuterResults
+
+Key | Description
+--- | ---
+status | `"complete"` (Can this take other values?)
+tests | array of `Test`
+
+## Test
+
+Key | Description
+--- | ---
+ccheck | ?
```

Comment:
I'm dropping this key, it's just to be used internally to decide if to skip cloudflares cache.